Lib/test/test_typing.py

```diff
@@ -491,7 +491,7 @@ def test_cannot_instantiate_vars(self):
 
     def test_bound_errors(self):
         with self.assertRaises(TypeError):
-            TypeVar('X', bound=Union)
+            TypeVar('X', bound=Optional)
```

Comment:
So passing `bound=Union` is still nonsensical, we just no longer diagnose it at runtime, right? (Nor do we diagnose e.g. `bound=42`, so I guess it's all right).